packages/astro/src/core/errors/errors-data.ts

```diff
@@ -493,7 +484,7 @@ export const ImageMissingAlt = {
 	name: 'ImageMissingAlt',
 	title: 'Missing alt property.',
 	message: 'The alt property is required.',
-	hint: "The `alt` property is important for the purpose of accessibility, without it users using screen readers or other assistive technologies won't be able to understand what your image is supposed to represent. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt for more information.",
+	hint: 'Use alt="" if image is not a key part of page content.',
```

Comment:
I commented already on it, but this lost a ton of information. This should be written from the perspective of not knowing what an `alt` tag is and why it's required.

 This edit ends up being frustrating to read due to the lack of info.

Reply:
From @Princesseuh (threading this here to keep the convo going in one place):

> In general, the entire purpose of the hints wasn't necessarily to be short, right. They're supposed to get users to potential solutions as soon as possible, so they should be concise (much like error messages), but not omit information that they can in fact provide.
> 
> Removing links from them to link to the error reference in addition of potentially delaying finding a solution, requires the user to do multiple context switches when we can link to docs page that explain the domain directly.
> 
> I don't personally see the value of reducing the length of errors that much. If I get an error, I'd rather the terminal be spammed with all relevant information than be pretty looking (and I'd much rather resize my terminal than visit multiple webpages)
> 
> We could argue that this new format makes it easier to ingest information, but I don't think that's true because of things being all together vs in different places. It just looks, subjectively, prettier.
> 
> (Edit, oops this was meant to be a reply to the comment, but for some reason GitHub Mobile put it top level)

Reply:
See longer response above: https://github.com/withastro/astro/pull/8833#discussion_r1367319517

I'm happy to work with you to improve any of these new hints, if we can keep my new breakdown of responsibility as "hints just hint, error reference URLs educate". I'm sure I err'd on the side of removing vs. keeping, so it makes total sense that some things should be added back.

+1 on adding back a note on why it's required. Let me give this a rewrite and then LMK what you think.

Reply:
Done! LMK WYT:

<img width="744" alt="Screen Shot 2023-10-20 at 11 29 07 AM" src="https://github.com/withastro/astro/assets/622227/79366545-97e5-4e08-8c6f-e8af15e3aeb5">

Reply:
> if we can keep my new breakdown of responsibility as "hints just hint, error reference URLs educate"

Why is this the new breakdown of responsibility? What is the motivation?